.gitlab-ci.yml

```diff
@@ -14,5 +14,5 @@ test:default:
       popd
     - mkdir build && cd build &&
       cmake -DCMAKE_PREFIX_PATH=$HOME/kokkos -DCabana_ENABLE_TESTING=ON -DCabana_ENABLE_EXAMPLES=ON .. &&
-      make -j2 VERBOSE=1 &&
+      make -k -O -j2 VERBOSE=1 &&
```

Comment:
the -0 looks to be causing issues in the travis build

Reply:
You are right, it is a make-4 feature, but Ubuntu’s make is older!